### ISSUE TITLE
Use `utils::package.skeleton` instead of `usethis::create_package` or (now defunct) `devtools::create`

### ISSUE THREAD

**Issue by russHyde** [russHyde/data_buddy#81]
For example, `devtools::create` has been moved to `usethis::create` as of `devtools`-v2.1.0; and `devtools::create` is used in `setup.DESCRIPTION.R`

Similarly,
- `usethis::use_package`
- `options(usethis.full_name = ...)`
- `usethis::use_testthat`

**Comment by russHyde** [comment]
NOOOOOO!!!!! Using usethis crashed my project management pipeline.

`usethis` won't allow project-specific package(s) to be nested inside a research project. {and seemingly won't ever allow this - https://github.com/r-lib/usethis/issues/553 ].

When trying to setup the skeleton for a package using usethis in my setup scripts, an interactive prompt attempts to load (and fails, because it is running in a non-interactive session, and there is no way to provide a `force_create = TRUE` argument that provides an answer to the interactive Query):

```
New project 'stemness' is nested inside an existing project './', which is rarely a good idea.
Error: User input required, but session is not interactive.
Query: Do you want to create anyway?
Execution halted
```

**Comment by russHyde** [comment]
Suggest rewriting the scripts to use `utils::package.skeleton`.

**Comment by russHyde** [comment]
Can use existing scripts provided devtools < 2.1.0 (since `devtools::create` was removed in devtools-2.1.0)

**Comment by russHyde** [comment]
Fixed in #91 (setup.DESCRIPTION.R is no longer used)

A local package skeleton is made using `utils::package.skeleton`; the user can modify this as they wish, and this package is kept under version control (in ./lib/local/<package.name>)

Devtools is used when building the package (but not during install or initialisation of the package); could actually use `pkgbuild` rather than `devtools` to reduce dependencies in `package_builder.R`
